Sources/OpenWeatherKit/Internal/Extensions/APIForecastHourly+Map.swift

```diff
@@ -31,6 +31,8 @@ extension APIHour {
             precipitationAmount: Measurement(value: precipitationAmount, unit: .millimeters),
             pressure: Measurement(value: pressure, unit: .millibars),
             pressureTrend: PressureTrend(rawValue: pressureTrend) ?? .undefined,
+            snowfallIntensity: Measurement(value: snowfallIntensity, unit: .millimetersPerHour),
+            snowfallAmount:Measurement(value: snowfallAmount, unit: .millimeters),
```

Comment:
Using values if available, otherwise default to 0.0.
Note: That I have not verified these values in the native framework, but seems reasonable enough.

```suggestion
            snowfallIntensity: snowfallIntensity != nil ? Measurement(value: snowfallIntensity!, unit: .millimetersPerHour) : Measurement(value: 0.0, unit: .millimetersPerHour),
            snowfallAmount: snowfallAmount != nil ? Measurement(value: snowfallAmount!, unit: .millimeters) : Measurement(value: 0.0, unit: .millimeters),
```